temporary-plugins/superset-ui-plugins-demo/storybook/stories/index.js

```diff
@@ -2,10 +2,10 @@ import '@babel/polyfill';
 import { setAddon, storiesOf } from '@storybook/react';
 import { withKnobs } from '@storybook/addon-knobs';
 import JSXAddon from 'storybook-addon-jsx';
-import categoricalD3 from '@superset-ui/color/esm/colorSchemes/categorical/d3';
-import sequentialCommon from '@superset-ui/color/esm/colorSchemes/sequential/common';
-import sequentialD3 from '@superset-ui/color/esm/colorSchemes/sequential/d3';
-import { configure } from '@superset-ui/translation';
+import categoricalD3 from '@superset-ui/color/lib/colorSchemes/categorical/d3';
+import sequentialCommon from '@superset-ui/color/lib/colorSchemes/sequential/common';
+import sequentialD3 from '@superset-ui/color/lib/colorSchemes/sequential/d3';
```

Comment:
We are not generating `esm` files any more, since the content is identical to `lib` (checked multiple random files).

Reply:
I dig a bit deeper and found that last time I modified babel config there was a mistake so both `esm` building and `commonjs (lib)` ended up pointing to same babel config so the `esm` output now is not correctly built. 

We should fix this so the `esm` output (not this PR, but just don't drop `esm`) will be different from `lib` and using `import/export` instead of `exports.xxx`. The `esm` output should be more preferred for webpack than `lib`. 

> So, what we've learned is that in order to take advantage of tree shaking, you must...
> * Use ES2015 module syntax (i.e. import and export).
> * Ensure no compilers transform your ES2015 module syntax into CommonJS modules (this is the default behavior > of the popular Babel preset @babel/preset-env - see the documentation for more details).
> * Add a "sideEffects" property to your project's package.json file.

from webpack doc https://webpack.js.org/guides/tree-shaking/#clarifying-tree-shaking-and-sideeffects

Reply:
Just pushed a fix in my branch: https://github.com/apache-superset/superset-ui/pull/729/commits/18b05b1345f50a8db21cf97b2d2dea83cedad2b9